internal/adapters/clients/ollama.go

```diff
@@ -11,110 +11,23 @@ import (
 	"github.com/sashabaranov/go-openai"
```

Comment:
Consider documenting what configuration is required for 'NewOllamaClient' function and what it initializes.